src/components/TodoTask.jsx

```diff
@@ -0,0 +1,38 @@
+import React, {useState} from 'react';
+import EditingTemplate from './EditingTemplate';
+import ViewTemplate from './ViewTemplate';
+
+export default function TodoTask({id, text, onDeleteTask, onEditTask}) {
+    
+    const [isEditing, setEditing] = useState(false);
```

Comment:
Convenția este ca numele setter-ului să fie numele state-ului cu prefixul `set`.

```suggestion
    const [isEditing, setIsEditing] = useState(false);
```